doc/_admin-guide/110_Template_and_rewrite/000_Customize_message_format/004_Macros_of_syslog-ng.md

```diff
@@ -438,19 +438,24 @@ This macro is available in syslog-ng OSE 4.5 and later versions. The syslog-ng O
   * rfc3164+tls
   * rfc3164+tcp
   * rfc3164+udp
+  * rfc3164+proxied-tcp
   * rfc3164+proxied-tls
+  * rfc3164+proxied-tls-passthrough
   * rfc3164+ \<custom logproto like altp\>
 
 * UNIX domain drivers `unix-dgram()`, `unix-stream()`
   * `unix-stream`
   * `unix-dgram`
 
 * rfc5424 style syslog `syslog()`:
+  * rfc5424+proxied-tls
```

Comment:
just curious, why not before the 

`rfc5424+proxied-tls-passthrough`

like in the previous case

Reply:
No particular reason, but you're right, it makes more sense there. Updated.